const handleError = (e) => console.log("ERROR HAS HAPPENED: ", e);

export default handleError;
